sky/clouds/service_catalog/images/provisioners/skypilot.sh

```diff
@@ -4,12 +4,17 @@
 sudo systemctl stop unattended-upgrades || true
 sudo systemctl disable unattended-upgrades || true
```

Comment:
Actually, might be a mistake in my previous PR, should it be `unattended-upgrades` or `unattended-upgrades.service`? Should we only keep one of them?

Reply:
It doesn't really matter - systemctl allows the .service suffix to be omitted for simplicity, and if you don't add .servce systemctl will automatically append .service to it behind the scenes